src/Object/Api/Friendica/Circle.php

```diff
@@ -23,12 +23,7 @@
 
 use Friendica\BaseDataTransferObject;
 
-/**
- * Class Group
- *
- *
- */
```

Comment:
Class documentation has been removed here.

Reply:
Yeah, it wasn't adding any meaningful details.